src/components/Button/Button.module.css

```diff
@@ -22,6 +22,11 @@
   justify-content: center;
   gap: var(--spacing-gap-xs, 4px);
   box-shadow: none;
+
+  & :global(.mia-platform-btn-icon) {
```

Comment:
Can you add a comment explaining this requirement?

Reply:
pretty straightforward: antd sets a padding based on the iconPosition but our button is already setting a gap